src/views/SessionView/Whiteboard.vue

```diff
@@ -142,7 +142,7 @@
         <input
           type="file"
           class="upload-photo"
-          accept="image/jpg, image/png"
+          accept="image/jpeg, image/png"
```

Comment:
Oh good catch! While we're at it, what about using `accept="image/*"`? Could you see if uploading other image types will work ok with zwibbler?

Reply:
Just tried using `accept=image/*`:

- Was able to upload an SVG!
- Uploaded a GIF, but only the first frame is shown, so it's basically converted to an image

Reply:
sweet! let's go with that then